src/tz.cpp

```diff
@@ -2739,53 +2748,26 @@ init_tzdb()
 {
     std::unique_ptr<tzdb> db(new tzdb);
 
-    //Iterate through folders
-    std::queue<std::string> subfolders;
-    subfolders.emplace(get_tz_dir());
-    struct dirent* d;
-    struct stat s;
-    while (!subfolders.empty())
-    {
-        auto dirname = std::move(subfolders.front());
-        subfolders.pop();
-        auto dir = opendir(dirname.c_str());
-        if (!dir)
+    for (const auto& filename : get_available_file_names()) //RTC Logic changed to account for windows and embedded data support
+    {
+        if (filename[0] == '.' ||
+            memcmp(filename.c_str(), "posix", 5) == 0 || // starts with posix
+            strcmp(filename.c_str(), "Factory") == 0 ||
+            strcmp(filename.c_str(), "iso3166.tab") == 0 ||
+            strcmp(filename.c_str(), "right") == 0 ||
+            strcmp(filename.c_str(), "+VERSION") == 0 ||
+            strcmp(filename.c_str(), "version") == 0 ||
+            strcmp(filename.c_str(), "zone.tab") == 0 ||
+            strcmp(filename.c_str(), "zone1970.tab") == 0 ||
+            strcmp(filename.c_str(), "tzdata.zi") == 0 ||
+            strcmp(filename.c_str(), "leapseconds") == 0 ||
+            strcmp(filename.c_str(), "leap-seconds.list") == 0 ||
+            strcmp(filename.c_str(), "windowsZones.xml") == 0)
```

Comment:
I couldn't find two ways about it. This code relies on Dirent, which wasn't suitable for windows. I've kept the logic of the dataloading the same, but unfortunately needed to nix the loading logic.

On the brightside this area of code is unlikey to experience much churn,